### ISSUE TITLE
C4解除後にアイコンが緑色に光るようにする

### ISSUE THREAD

**Issue by ruk1ium** [Thiry1/csgo-custom-hud#20]
C4解除完了後、次のラウンドが始まるまで
画面上のC4アイコンを緑色に光らせると見栄えが良くなると思います！

[IEMHUDの例](https://www.twitch.tv/videos/233326359?t=07h46m02s)

**Comment by LukasW1337** [comment]
Agree! This would make it more fluently transition and people will still know last round was a bomb Defuse Win
